packages/harmony/src/components/completion-check/CompletionCheck.module.css

```diff
@@ -0,0 +1,40 @@
+.container {
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  width: 16px;
+  height: 16px;
+}
+
+.defaultCompletionIcon {
+  position: absolute;
+  display: inline-flex;
+  align-items: center;
+  height: 12px;
+  width: 12px;
+  border: 2px solid var(--harmony-neutral);
+  border-radius: 50%;
+}
+
+.completionIcon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  border-radius: 50%;
+  z-index: 15;
+}
+
+.emptyCompletionIcon {
+  height: 18px;
+  width: 18px;
+}
+.checkIcon,
+.removeIcon {
+  z-index: 15;
+}
+
+.checkIcon > svg path,
+.removeIcon > svg path {
+  fill: var(--harmony-white);
```

Comment:
just wanted to point out that `--harmony-white` is a variable color which changes based on the theme and is not actually white in dark mode. `--harmony-static-white` will always be `#FFFFFF`